PhysicsTools/NanoAOD/python/V10/photons_cff.py

```diff
@@ -13,6 +13,7 @@
         'RecoEgamma.PhotonIdentification.Identification.mvaPhotonID_Fall17_94X_V2_cff',
```

Comment:
Please revert the changes to the V10 config. The V10 is frozen and should not be modified.

Reply:
The WF 2500.6 was calling the V10 nanoAOD and hence, crashed since it was calling the /V10/photons_cff.py file. I changed the V10 because of that. Should I revert back the changes then?

Reply:
I have revert back the changes in the V10 nanoAOD and made changes to `IsoValueMapProducer` so, that quadratic EA correction is optional.